pytd/writer.py

```diff
@@ -368,6 +370,11 @@ def write_dataframe(
             Each type of element of list will be converted by
             ``numpy.array(your_list).tolist()``.
 
+        show_progress : boolean, default: False
+            If this argument is True, shows a TQDM progress bar
+            for chunking data into msgpack format and uploading before
+            performing a bulk import.
+
             If True, ``fmt`` argument will be overwritten with ``msgpack``.
```

Comment:
This document should be under the `keep_list`.

Reply:
Whoops, good catch - accidentally pasted it above